CHANGELOG.md

```diff
@@ -1,3 +1,169 @@
+# #develop#
+
+## Breaking changes
+
+## New deprecations
+
+* Deprecated `MDCAppBarTextColorAccessibilityMutator`.
+
+### FlexibleHeader
+
+* [minMaxHeightIncludesSafeArea](https://github.com/material-components/material-components-ios/issues/4764)
```

Comment:
This is not yet deprecated.

Can you add a "## To be deprecated" section for APIs that will be deprecated soon? We'll need to update the template accordingly as well.